store.js

```diff
@@ -1,3 +1,65 @@
+'use strict';
 //individaual stores become sparate objects
 /*first colum is location second colum is hours third columb is expected customers
 and fouthr colum is expected cookies per hour fifth colum is totol cookies in day*/
+var stores = [
```

Comment:
You shouldn't need to create a new `stores` list, since `model.js` is being loaded first. You can use the existing `listOfStores`.